conda_build/skeletons/rpm.py

```diff
@@ -0,0 +1,668 @@
+import argparse
```

Comment:
Please include at least one test of this.  Skeletons are where we have the worst coverage, and I don't want to keep up that trend.